vm/src/rv32/parse.rs

```diff
@@ -210,8 +210,8 @@ pub(crate) fn parse_u32(word: u32) -> Option<RV32> {
         OPC_FENCE => FENCE,
 
         _ => match word {
-            0x00000073 => ECALL,
-            0x00100073 => EBREAK,
+            0x00000573 => ECALL { rd: rd(word) },
+            0x00100573 => EBREAK { rd: rd(word) },
```

Comment:
I'm still digesting the code, so bear with me. I follow how this decodes the `RV32Nexus` forms of `ECALL` and `EBREAK` when `rd` is `10` (decimal), but is the intention to support all possible values of `rd` encoded in bits 7-11 (inclusive)?

I suppose as long as the only supported inputs are rv32i-encoded instructions and we control the mapping (`translate_nexus()` below), then we're the only direct user of this RV32Nexus ISA.

Assuming fixing `rd := 10` is sufficient for our needs, what do you think about either ...

1. Restricting the RV32Nexus EXTENSION to only support `rd` encoded as `10` for now or
2. Reverting to `ECALL` and `EBREAK` variants not having any associated data and merely changing their semantics in RV32Nexus EXTENSION to change where/how they return.

Reply:
Good call, will generalize, even if the VM will always use `rd = 10` for now.

Reply:
Should be resolved now.